src/test/kotlin/io/perfometer/statistics/ScenarioSummarySpecification.kt

```diff
@@ -78,61 +78,85 @@ class ScenarioSummarySpecification {
     @Test
     fun `should return request times greater or equal than 9Xth percentile of requests`() {
         val stats = (1..100).map {
-            RequestStatistics("GET", HttpMethod.GET, "", Duration.ofMillis(it.toLong()), ok)
+            RequestStatistics("GET", HttpMethod.GET, "", Instant.ofEpochMilli(it.toLong()), Instant.ofEpochMilli((2 * it).toLong()), ok)
         }
 
         val totalSummary = ScenarioSummary(stats, start, end).totalSummary
-        totalSummary.shouldNotBeNull()
-                .percentile95Time shouldBe Duration.ofMillis(95)
+        totalSummary
+            .shouldNotBeNull()
+            .percentile95Time shouldBe Duration.ofMillis(95)
         totalSummary.percentile96Time shouldBe Duration.ofMillis(96)
         totalSummary.percentile97Time shouldBe Duration.ofMillis(97)
         totalSummary.percentile98Time shouldBe Duration.ofMillis(98)
         totalSummary.percentile99Time shouldBe Duration.ofMillis(99)
     }
 
+    @Test
+    fun `should calculate requests per second`() {
+        val stats = (0 until 1000).map {
+            RequestStatistics("GET", HttpMethod.GET, "", Instant.ofEpochMilli(10 * it.toLong()), Instant.ofEpochMilli((10 * it).toLong()), ok)
```

Comment:
Consider using a call to StatisticsFixture instead of manually creating RequestStatistics instance